packages/hofexpress_app/Configuration/TypoScript/setup.typoscript

```diff
@@ -1,4 +1,4 @@
-plugin.tx_hofexpressapp_hofexpress {
+plugin.tx_hofexpressapp {
```

Comment:
Actually that was the cause of TypoScript page setting not being available in the new `OrderPlugin` plugin...

* `tx_hofexpressapp_hofexpress` defines settings just for `Hofexpress` plugin (first one in `ext_localconf.php`)
* `tx_hofexpressapp_orderplugin` would have defined settings for new `OrderPluging` plugin
* `tx_hofexpressapp` is simpler and now defined settings for *ALL* plugins